### PR TITLE
Fix #3129 cash rec form aligment

### LINKED ISSUE #3129: UI Issue Report - Cash Receipts by Provider
### Describe the bug

UI Issue Report Cash Receipts by Provider form field alignment.

### To Reproduce

1. Choose Report > Financial
2. Click Cash Rec
3. See the form field

image

### Client configuration

- Browser: Firefox 73.0.1（64 bit）
- Operating system: Windows 10

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #3129 

#### Short description of what this resolves:
Remove unnecessary string.
Move the title header and form to the Bootstrap container class.

image

### CODE REVIEW

**Comment by stu01509** [comment]
Hi @bradymiller, @tywrenn 

Do you recommend me to refactor this page, I think such this page layout, It used a lot of nested tables to design and include many outdated or invalid syntax and difficult to maintain.
So I would like to use the Bootstrap grid system and table to improve it. Is a good idea?

**Comment by sjpadgett** [comment]
I recom lose the table if possible however, remember that in reports tables are very handy for printing. I've had pretty good results with printing BS grid in past. Just depends on the report.

**Comment by stu01509** [comment]
Hi @sjpadgett 

Thanks for your reply and suggestion, How do you think I use Bootstrap grid system to replace the table tag layout, and remove inline-style, Is it possible to happen the printing error or overflow?

image
sl_receipts_report.php

File: interface/billing/sl_receipts_report.php [hunk 1]
```diff
@@ -156,584 +156,598 @@ function sel_diagnosis() {
...
+    <div class="row">
+        <div class="col-12">
+               <form method='post' action='sl_receipts_report.php' id='theform' onsubmit='return top.restoreSession()'>
+                <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+
+                <div id="report_parameters">
+
+                <input type='hidden' name='form_refresh' id='form_refresh' value=''/>
+
+                <table>
```
Comment (tywrenn) [inline interface/billing/sl_receipts_report.php]: This entire table needs reformatting

File: interface/billing/sl_receipts_report.php [hunk 2]
```diff
@@ -156,584 +156,598 @@ function sel_diagnosis() {
...
-
-     <tr bgcolor="#ddddff">
-        <td class="detail" colspan="<?php echo ($form_proc_codefull ? 4 : 2) + ($form_procedures ? 2 : 0); ?>">
-                    <?php echo xlt('Totals for ') . text($docname) ?>
-  </td>
-  <td align="right">
-                    <?php bucks($doctotal1) ?>
-  </td>
+                <div id="report_results">
+                <table class='table' cellpadding='1' cellspacing='2' width='98%' id='mymaintable'>
```
Comment (tywrenn) [inline interface/billing/sl_receipts_report.php]: So does this one

File: interface/billing/sl_receipts_report.php [hunk 3]
```diff
@@ -156,584 +156,598 @@ function sel_diagnosis() {
...
+                                $amount1 -= $row['amount'];
+                            }
+
+                        // if ($docid != $row['employee_id']) {
+                            if ($docid != $row['docid']) {
+                                if ($docid) {
+                                    // Print doc totals.
+                                    ?>
+
+                    <tr bgcolor="#ddddff">
```
Comment (tywrenn) [inline interface/billing/sl_receipts_report.php]: bgcolor shouldn't exist anymore

File: interface/billing/sl_receipts_report.php [hunk 4]
```diff
@@ -156,584 +156,598 @@ function sel_diagnosis() {
...
+                        <?php bucks($doctotal1) ?>
+                </td>
+                        <?php if ($form_procedures) { ?>
+                <td align="right">
+                            <?php bucks($doctotal2) ?>
+                </td>
+                <?php } ?>
+                </tr>
+
+                <tr bgcolor="#ffdddd">
```
Comment (tywrenn) [inline interface/billing/sl_receipts_report.php]: bgcolor shouldn't exist anymore

**Comment by stu01509** [comment]
Hi @tywrenn , @bradymiller 

Thanks for your review.

I have a suggestion, I think we may need to discuss the OpenEMR standard layout.

In the current project,  OpenEMR has a lot of different layouts, some pages contain outdated syntax, some pages didn't use the Bootstrap and some pages have a strange layout, I think we should planing a design guideline about OpenEMR, it will be easy for our to refine the pages.

I list some pages of the screenshot.
image

image

image

image

image

image

**Comment by bradymiller** [comment]
hi @stu01509 , @im-Amitto , and @tywrenn . A styling guide sounds like a good idea. We could place it here in place of this very outdated doc: https://github.com/openemr/openemr/blob/master/interface/README.md

Would it make sense to first point out pages where the styling is what we want (ie. point out the good examples) and then go from there?

**Comment by stu01509** [comment]
Hi @bradymiller,

I am so glad you agree with my idea, I also agree to place the styling guide in
https://github.com/openemr/openemr/blob/master/interface/README.md

I think we can create a new issue to discuss the styling guide, and complete this PR,
because I think to plan the styling guide it would take a lot of time.

**Comment by im-Amitto** [comment]
@stu01509 Nice idea, We do need a styling guide as most pages doesn't have even basic bootstrap style e.g. `col-`, `row-`, `container` etc.

**Comment by bradymiller** [comment]
hi @stu01509 and @im-Amitto , Sounds like a good plan
@stu01509 , Do you want me just bring in this PR (only minor changes in this PR and makes sense to just bring this in now if you want).

**Comment by stu01509** [comment]
Hi @bradymiller,

Please bring in this PR, Thanks.

**Comment by bradymiller** [comment]
hi @stu01509 , done :)

**Pull request merged**